apps/crn-server/serverless.ts

```diff
@@ -58,6 +56,12 @@ const envRef =
     : `CI-${SLS_STAGE}`;
 const sentryDsnApi = CRN_SENTRY_DSN_API!;
 const sentryDsnHandlers = CRN_SENTRY_DSN_HANDLERS!;
+if (
+  (SLS_STAGE === 'dev' || SLS_STAGE === 'production') &&
+  !(sentryDsnApi && sentryDsnHandlers)
+) {
+  assert.ok('sentry DSN API or Handler not defined');
+}
```

Comment:
I think you want to do this instead
```suggestion
if (SLS_STAGE === 'dev' || SLS_STAGE === 'production') {
  ['CRN_SENTRY_DSN_API', 'CRN_SENTRY_DSN_HANDLERS'].forEach(env) => {
  assert.ok(process.env[env], `${env} not defined`);
});
}
```
and before the assignment too